pkgs/by-name/rp/rp2040js/package.nix

```
@@ -0,0 +1,77 @@
+{
+  fetchFromGitHub,
+  lib,
+  makeWrapper,
+  nix-update-script,
+  testers,
+
+  buildNpmPackage,
+  typescript,
+  nodejs,
+  ...
+}:
+let
+  version = "1.1.1";
+in
+buildNpmPackage (finalAttrs: {
+  pname = "rp2040js";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "wokwi";
+    repo = "rp2040js";
+    tag = "v${version}";
```

Suggestion, you don't have to do this

```suggestion
buildNpmPackage (finalAttrs: {
  pname = "rp2040js";
  version = "1.1.1";

  src = fetchFromGitHub {
    owner = "wokwi";
    repo = "rp2040js";
    tag = "v${finalAttrs.version}";
```